emily/handler/src/api/handlers/chainstate.rs

```diff
@@ -104,29 +132,14 @@ pub async fn set_chainstate(
         context: EmilyContext,
         body: SetChainstateRequestBody,
     ) -> Result<impl warp::reply::Reply, Error> {
-        // Create new chainstate table entry.
-        let new_chainstate_entry: ChainstateEntry = ChainstateEntry {
-            key: ChainstateEntryKey {
-                height: body.stacks_block_height,
-                hash: body.stacks_block_hash,
-            },
-        };
-
-        // TODO(TBD): Tactfully handle state inconsistencies and race conditions across
-        // multiple requests.
-        //
-        // - chain tip is ahead.
-        // - chain tip is far behind.
-        // - two chain tips are proposed at the same time for conflicting hashes
-        // - etc.
-        //
-        // `get_chain_tip_or_set_if_absent` is inappropriate in the long run and should be
-        // deleted.
-
-        accessors::add_chainstate_entry(&context, &new_chainstate_entry).await?;
-        let response: Chainstate = new_chainstate_entry.into();
+        // Convert body to the correct type.
+        let chainstate: Chainstate = Chainstate::from(body);
```

Comment:
Now `clippy` is complaining about a useless conversion 😞. Okay, well just do what you want here.

Reply:
Reverted to old version :(